### PR TITLE
Awesome-books-ES6-modules

### PR DESCRIPTION
**Pull request by Ibizugbe**
I have created the Awesome books app in line with the project specifications.
-  I made sure there were no linter errors.
-  I ensured I used the correct GitHub flow.
-  I ensured my work was documented professionally.
-  I ensured I used best practices in HTML, CSS, and JavaScript.
-  I used ES6 syntax.
-  I kept all functionalities of the app without errors.
- I divided my code into modules and kept each functions in a separate file.

### CODE REVIEW

File: modules/local_storage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export default function storageAvailable(type) {
+  // Check if Storage is available
+  let storage;
```
Comment (julie-ify) [inline modules/local_storage.js]: - [ ] Kindly make use of the ES6 arrow syntax function instead of the function keyword.

File: README.md [hunk 2]
```diff
@@ -1,62 +1,76 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Awesome Books ES6
 
+> Awesome books is an app where you can add and remof ove list books you are currently reading. This app is built with the latest ES6 technology.
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly fix the typographical error here, you have `remof ove` instead of `remove`.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Ibizugbe,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vechicin — APPROVED** [review]
# Hi @Ibizugbe ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good job implementing the previous reviewer's requirements
- [x] Excellent UI!

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**